ProjectGroupA/src/main/java/com/fundation/search/View/ActionSearch.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  @(#)AssetFile.java Copyright (c) 2018 Jalasoft.
+ *  2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ *  All rights reserved.
+ *  <p>
+ *  This software is the confidential and proprietary information of
+ *  Jalasoft, ("Confidential Information").  You shall not
+ *  disclose such Confidential Information and shall use it only in
+ *  accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.fundation.search.View;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Map;
+
+public class ActionSearch implements ActionListener{
```

Comment:
Need a description for the class and this might need to go to the controller right?

Reply:
this class will be removed next commit